string label
float32[3] position
float32 center
float32 distance
